nautobot_circuit_maintenance/models.py

```diff
@@ -239,38 +235,31 @@ def to_csv(self):
 class NotificationSource(OrganizationalModel):
     """Model for Notification Source configuration."""
 
-    # Mark field as private so that it doesn't get included in ChangeLogging records!
-    _password = encrypt(models.CharField(max_length=100))
-    source_id = models.EmailField(max_length=100, unique=True, help_text="Email address used as identifier.")
-    url = models.CharField(max_length=200)
-    source_type = models.CharField(
-        default=NotificationSourceServerChoices.UNKNOWN,
-        max_length=50,
-        choices=NotificationSourceServerChoices,
-        null=True,
-        blank=True,
-        help_text="Type of source integration to retrieve the notifications.",
+    name = models.CharField(
+        max_length=100,
+        unique=True,
+        help_text="Notification Source Name as defined in configuration file.",
     )
-
+    slug = models.SlugField(max_length=100, unique=True)
     providers = models.ManyToManyField(
         Provider,
-        help_text="The Provider(s) to which this notification source applies.",
+        help_text="The Provider(s) that this Notification Source applies to.",
         blank=True,
     )
 
-    csv_headers = ["source_id", "_password", "url"]
+    csv_headers = ["name", "slug"]
```

Comment:
Based on `to_csv` below, I'd think this should be:
```suggestion
    csv_headers = ["name", "slug", "providers"]
```